doc/user/content/guides/cdc-postgres.md

```diff
@@ -0,0 +1,260 @@
+
+1. Grant the required privileges to the replication user:
+
+    ```sql
+    ALTER ROLE "user" WITH REPLICATION;
+    ```
+
+    **Note:** This user also needs `SELECT` privileges on the tables you want to replicate, for the initial table sync.
+
+1. Set the replica identity value for the tables you want to replicate:
+
+    ```sql
+    ALTER TABLE repl_table REPLICA IDENTITY FULL;
+    ```
+
+    This setting determines the amount of information that is written to the WAL in case of `UPDATE` and `DELETE` operations. By setting it to `FULL`, change events will include the previous values of all the table’s columns; as a heads-up, you should expect a performance hit in the database from increased CPU usage.
+
+1. Create a [publication](https://www.postgresql.org/docs/current/logical-replication-publication.html) with the tables you want to replicate:
+
+    _For specific tables:_
+
+    ```sql
+    CREATE PUBLICATION mz_source FOR TABLE table1, table2;
+    ```
+
+    _For all tables in Postgres:_
+
+    ```sql
+    CREATE PUBLICATION mz_source FOR ALL TABLES;
+    ```
+
+    The `mz_source` publication contains the set of change events generated from `table1` and `table2`. This publication will later be used to ingest the replication stream into Materialize.
+
+### Create a source
+
+Postgres sources ingest the raw replication stream data for all tables included in a publication to avoid creating multiple replication slots and minimize the required bandwidth, and must therefore be explicitly materialized. To create a source:
+
+```sql
+CREATE MATERIALIZED SOURCE mz_source
+FROM POSTGRES
+  CONNECTION 'host=example.com port=5432 user=host dbname=postgres sslmode=require'
+  PUBLICATION 'mz_source';
+```
+
+{{< note >}}
+Materialize performs an initial sync of the tables included in the publication before it starts ingesting change events, so you should make sure that your source tables **fit into memory**.
+{{</ note >}}
+
+In practice, the `mz_source` looks like:
+
+```sql
+SHOW COLUMNS FROM mz_source;
+
+   name   | nullable |  type
+----------+----------+---------
+ oid      | f        | integer
+ row_data | f        | list
+```
+
+Each row of every upstream table is represented as a single row with two columns in the replication stream source:
+
+* `oid`: A unique identifier for the tables included in the publication;
+
+* `row_data`: A text-encoded, variable length `list`. The number of text elements in a list is always equal to the number of columns in the upstream table.
+
+The next step is to break down this source into views that reproduce the publication’s original tables and can be used as a base for your materialized views.
```

Comment:
I'd insert a "that" before "can" (not required, stylistic preference). This is a little confusing, though,  because we already said the source is materialized; we should say why the views need to be materialized on top of that.